test/.eslintrc

```diff
@@ -0,0 +1,22 @@
+env:
+  mocha: true
+
+plugins:
+  - mocha
```

Comment:
Why does this get a `-`, but the ones in `rules` do not?

(I was cursing at YML last night)